client/html/pages/panel/jobs/show.html

```diff
@@ -77,7 +77,29 @@ <h3 class="text-primary">Tags</h3>
                       {{/each}}
                       </ul>
                     </div>
+
```

Comment:
I think it's better if we publish all students in a specific page. For now, we don't need students to see other students. Can you make it in a specific page where only companies can access? Thanks